src/greatwall/gui.py

```diff
@@ -60,7 +68,166 @@ def cancel(self):
         self.canceled.emit()
 
 
-class GreatWallQt(QMainWindow):
```

Comment:
Where is it?

Reply:
Its name has been changed to `GreatWallGui` and the implementation moved to line 230 in the same file.